www/source/docs/run-packages-export.html.md

```diff
@@ -21,6 +21,9 @@ You can create a Docker container image for any package by performing the follow
 
 5. You can now exit the studio. The new Docker container image exists on your computer and can be examined with `docker images` or run with `docker run`.
 
+
+Note that the `hab pkg install <local artifact>` command will only work if the studio has a copy of the origin's public key. If the install command fails with a 404 on the origin in question, make sure the public depot has a copy of the origin's public key.
```

Comment:
This sounds like more a FAQ "I keep seeing 404 when i try to install a package, what is going on?"

Another thing to note is that the workaround is not to have to depend on the public depot.  As long as you have a way to distribute the pub key that was used during building of the hart, then is the pub key that needs to be ensured that is in the studio or whenever the install needs to happen.  More or less, if you distribute the hart, you need to distribute the pub key that is associated with it otherwise Hab attempts to locate that info from the depot.